powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/PowerAuthRequestWrapper.java

```diff
@@ -25,23 +25,23 @@
  * @param <T> Type of the request object instance.
  * @author Petr Dvorak, petr@wultra.com
  */
-public class RESTRequestWrapper<T> {
+public class PowerAuthRequestWrapper<T> {
```

Comment:
This is probably a leftover from the past - can we use the [`ObjectRequest`](https://github.com/wultra/lime-java-core/blob/develop/rest-model-base/src/main/java/io/getlime/core/rest/model/base/request/ObjectRequest.java) class? Or is this because of the JavaX Validation annotation?

Reply:
Yes, it seems like we can easily replace the wrappers with `ObjectRequest`/`ObjectResponse`. I though that there was some extra functionality, but there is not.